### ISSUE TITLE
BiglyBT reinstalls on ever container change

### ISSUE THREAD

**Issue by Snake883** [Fullaxx/biglybt#10]
Every time I make a change to the container configuration via Unraid docker management, the container is removed and reinstalled, , and I also lose my configuration.  

When I remote into the container, I don't see any files being stored to the container path /config.
In the [documentation](https://hub.docker.com/r/fullaxx/biglybt/), it looks like the volume mapping for /config is wrong. I think the config is stored in the root folder, so I mapped the root container volume to my host to persist most of the configuration.  However...

I think when the container is recreated, it overwrites the biglybt.config file (or moves the previous file to biglybt.config.bak).  
I also lose the plugins I installed.
What might be causing this problem?  

Thank you

**Comment by Fullaxx** [comment]
Nothing is saved to /config from the container.
/config is for passing in biglybt.config or openvpn files to the container from the host.
/config has nothing to do with the .biglybt dir where BiglyBT keeps its preferences and running state.

Look for the section "Run the image keeping your biglybt configuration"
and use volume mapping to save your BiglyBT configuration:
-v /srv/docker/biglybt/data:/root/.biglybt

If this does not solve your issue,
paste in your docker run command or docker-compose file

**Comment by Snake883** [comment]
This worked:

> Look for the section "Run the image keeping your biglybt configuration"
> and use volume mapping to save your BiglyBT configuration:
> -v /srv/docker/biglybt/data:/root/.biglybt

Thank you for explaining /config.  Other popular containers/apps/maintaners (e.g. linuxserver.io) differ on how /config is used.

I also found that plugins can be installed for all users or current user, and the location of the plugin files are stored in another location.  It worked best/easier when I installed plugins for current user (not all users).
